docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,196 @@
+Installing and Using the XRootD Client 
+==============================================
 
-Install XRootd Client 
-=====================
+XRootD is a high performance data system widely used by several science VOs on OSG to store and to distribute data to
+jobs.
+It can be used to create a data store from distributed data nodes or to serve data to systems using a distributed
+caching architecture.
+Either mode of operation requires you to install the XRootD client software.
 
-XRootD is a high performance network storage system widely used in high energy physics experiments such as ATLAS and ALICE. The underlying XRootD data transfer protocol provides highly efficient access to ROOT based data files.
+This page provides instructions for installing and using the XRootD client software to access data using XRootD.
 
-This page provides instructions for using a XRootD client to connect to a XRootD storage system.
+As a user you have three different ways to interact with XRootD: 
 
-Installing a XRootD Client
---------------------------
+1. Using the XRootD client tools command 
+1. Using LD\_PRELOAD to use XRootD libraries with Unix tools
+1. Using a XRootDFS FUSE mount to mount a local XRootD data store
```

Comment:
Let's put the FUSE mount method before LD\_PRELOAD

Reply:
Done